src/components/menu/main-menu.tsx

```diff
@@ -110,86 +191,102 @@ const MenuItem = ({id, url, title, activeTrail, children, level}: MenuItemProps)
-      "border-digital-red lg:border-black": level === 0 && isCurrent,
-      "border-transparent lg:border-foggy-dark": level === 0 && !isCurrent && inTrail,
+      "lg:border-l-0 lg:border-b-[4px] lg:mr-0 lg:pb-10": level === 0,
+      "border-spirited-light": level === 0 && isCurrent,
+      "border-transparent lg:border-spirited-light":
+        level === 0 && !isCurrent && inTrail,
       "border-transparent": level === 0 && !isCurrent && !inTrail,
+      "lg:pr-10" : level === 0 && children.length === 0,
     },
     // Child menu item styles.
     {
       "ml-5 lg:ml-0 lg:pl-5": level !== 0,
-      "border-digital-red": level !== 0 && isCurrent,
-      "border-transparent": level !== 0 && !isCurrent
+      "border-spirited-light": level !== 0 && isCurrent,
+      "border-transparent": level !== 0 && !isCurrent,
     }
-  )
+  );
+
+  const chevronBtnStyles = clsx(
+    "shrink-0 relative pr-10 lg:right-0 text-black bg-transparent lg:bg-transparent lg:pr-5 lg:pb-6 lg:pl-2 lg:group-focus:border-b lg:group-hover:border-b-[4px] lg:group-focus:border-spirited-light lg:group-hover:border-spirited-light",
+    // Top menu item styles.
+    {
+      "border-b-[4px]": level === 0,
+      "border-spirited-light": level === 0 && isCurrent,
+      "border-transparent lg:border-spirited-light":
+        level === 0 && !isCurrent && inTrail,
+      "border-transparent": level === 0 && !isCurrent && !inTrail,
+    },
+  );
 
   const subMenuStyles = clsx(
-    "list-unstyled w-full min-w-[300px] lg:bg-white lg:shadow-2xl px-0 lg:absolute",
+    "list-unstyled w-full min-w-[350px] lg:bg-fog-light lg:border lg:border-white lg:shadow-2xl lg:absolute lg:rounded-[25px] lg:px-12 lg:py-4 lg:mt-3",
     zIndexes[level],
     {
       "lg:top-full lg:right-0": level === 0,
       "lg:top-0": level !== 0,
       "lg:left-full": level !== 0 && positionRight,
       "lg:right-full": level !== 0 && !positionRight,
-      "block": submenuOpen,
-      "hidden": !submenuOpen,
+      block: submenuOpen,
+      hidden: !submenuOpen,
     }
-  )
+  );
 
   return (
     <li
       ref={menuItemRef}
-      className={clsx("m-0 py-2 lg:py-0 relative border-b first:border-t last:border-0 border-cool-grey lg:border-black-20 lg:relative lg:mr-5 last:lg:mr-0", {"lg:border-b-0 first:border-t-0": level === 0})}
+      className={clsx(
+        "m-0 lg:py-0 relative border-b last:border-0 border-spirited-light lg:relative",
+        { "lg:border-b-0 first:border-t-0 lg:mr-10 lg:last:mr-0": level === 0 }
+      )}
     >
-      <div className="flex items-center justify-between lg:justify-end">
+      <div className="group flex lg:justify-end">
         <Link
           id={linkId}
           href={url || "#"}
           className={linkStyles}
           aria-current={isCurrent ? "true" : undefined}
         >
           {title}
+          {level !== 0 && <ArrowRightIcon width={20} className="hidden group-focus:inline-block group-hover:inline-block ml-2" />}
```

Comment:
does `group-focus` do anything since we can't actually focus on the `div`? I feel the `group` should probably be on the link element. This would also avoid adding the arrow when someone hovers over the side button.

Reply:
Ooo, great catch! I've removed the group-focus from the Chevron button. We'll keep the group-hover for visually users since that is the desired experience outlined in the mockup, unless it is going to cause accessibility issues. Let me know! 👀